lib/cn/wbt_internal.h

```diff
@@ -63,6 +63,8 @@ wbt_lfe_kmd(const struct wbt_node_hdr_omf *node, const struct wbt_lfe_omf *lfe)
         const uint32_t *p = (void *)node + omf_lfe_koff(lfe);
 
         kmd_off = omf32_to_cpu(*p);
+        memcpy(&kmd_off, (void *)node + omf_lfe_koff(lfe), sizeof(kmd_off));
```

Comment:
should we remove the previous 2 lines?

Reply:
Yeah, good catch!  Looks like my diff (~gbecker/align.diff didn't apply correctly...

Reply:
whoops. fixing